fml/platform/android/cpu_affinity.cc

```diff
@@ -20,6 +22,9 @@ namespace fml {
 std::once_flag gCPUTrackerFlag;
 static CPUSpeedTracker* gCPUTracker;
 
+std::once_flag gCheckManufacturerFlag;
```

Comment:
Nit: You should be able to put both these as statics in the `SetUpCPUTracker` method without polluting the namespace.